client/my-sites/customer-home/cards/notices/reader-first-posts/style.scss

```diff
@@ -1,4 +1,5 @@
 .reader-first-posts__nudge {
+	border-radius: 3px; /* stylelint-disable-line scales/radii */
```

Comment:
Hello @sixhours ! :) somehow this is making my build to fail:

```
client/my-sites/customer-home/cards/notices/reader-first-posts/style.scss
 ✖  Needless disable for "scales/radii"  --report-needless-disables
```

Any idea on how to proceed? I had this kind of issue before and I did something like this:

```
$custom-border-corner-size: 3px;

border-radius: $custom-border-corner-size;
```

Is it ok if I modify that code with the workaround?

Reply:
PS: https://github.com/Automattic/wp-calypso/pull/83372/files#diff-4dfc2d7d82572f19aa7a5341c64c97cc42c8d562ec84313be45f7e630ccf6d51R1-R3

Reply:
Looks like we can remove it like here https://github.com/Automattic/wp-calypso/pull/83365, pr here: https://github.com/Automattic/wp-calypso/pull/83385